index.html

```diff
@@ -3678,6 +3685,90 @@ <h3>Data Validation</h3>
 							</details>
 						</li>
 
+						<li id="validate-structural-relations">
+							<p>(<a href="#structural-rel"></a>) Verify the use of structural relations as follows:</p>
+							<ol>
+								<li id="rel-booleans">
+									<p>Let <var>contents</var>, <var>pagelist</var> and <var>cover</var> be <a
+											href="https://infra.spec.whatwg.org/#boolean">boolean</a> values set to
+										false.</p>
+								</li>
+								<li id="rel-create-list">
+									<p>Let <var>resources</var> be the result of <a
+											href="https://infra.spec.whatwg.org/#list-extend">extending</a>
+										<var>data["readingOrder"]</var> with <var>var["resources"]</var>.</p>
+								</li>
+								<li id="rel-iterate-list">
+									<p><a href="https://infra.spec.whatwg.org/#list-iterate">For each</a>
+										<var>resource</var> of <var>resources</var>, if <var>resource["type"]</var> is
+										set:</p>
```

Comment:
Here and in the rest of this step it should say <var>resource["rel"]</var> and not <var>resource["type"]</var> :-)

Reply:
wow, can't even blame that on a typo. my mind was elsewhere...